apps/web/app/modules/Statistics/utils.ts

```diff
@@ -0,0 +1,13 @@
+type Data =
```

Comment:
we need to add `object` to the union just to satisfy typescript, which is not ideal solution but it will work until we figure out how to generate better types from typebox schema